### ISSUE TITLE
Link to bios mod not working

### ISSUE THREAD

**Issue by bishwos** [stevenmirabito/T450s-Hackintosh#1]
Hey Steven,

Thank you for the guide. I am planning to install hackintosh in my system using your guide.

But the link for the bios file doesn't seem to be working. And I couldn't find another one when I googled it. Could you re-upload it.

Thanks,

**Comment by polarlights** [comment]
U can patch the AppleIntelBDWFrameBuffer with clover. [FYI](https://github.com/RehabMan/OS-X-Clover-Laptop-Config)

**Comment by stevenmirabito** [comment]
@bishwos Sorry, I didn't see this issue until just now! I've been meaning to revisit this guide, especially the parts related to the BIOS mod. I'll take a look and see if I can find the necessary files you will need to patch your own firmware image (as the raw dump contains MAC addresses and your Windows key).

@polarlights That option does exist, and it's certainly worth a try, but I tried several variations of Clover patches and none worked for me.

**Comment by polarlights** [comment]
@stevenmirabito  Yeah, it didn't work for me when I installed the 10.12. But after I installed it, it worked. I don't know why. Maybe I have some typo errors.  I used to replace the content of the framer, repair the permissions.

However I replaced the default the wireless card with the bcm94352z by modifying the BIOS mod, It works well. But the bluetooth doesn't work well, although it works great in 10.11.

**Comment by bhagatparwinder** [comment]
@stevenmirabito I have the raw bin dump for the BIOS but I cannot find the offset or areas that do need patching to unlock the advanced menu. Do you have those details or are you planning to add them (considering this issue is about 4 years old)?

**Comment by step21** [comment]
@stevenmirabito @bhagatparwinder if either of you have an umodded bios, that would already help. (my nvram got messed up when installing) I have the FL1 file from the lenovo update but it's smaller but seemingly not compressed so according to some online comments, I would have to copy the good parts over my backup of the bios backup to end up with a flashable file.

**Comment by step21** [comment]
Update: fixed it by flashing a full bios region from an almost identical machine.
